content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -4055,72 +4183,51 @@ Informs that a chat thread was untagged.
 
 ## Customers
 
-### `customer_visit_started`
+### `incoming_customers`
 
-Informs that a Customer entered the tracked website.
+Informs about customers the agent should be aware of. It's sent after [`login`](#login).
```

Comment:
```suggestion
Informs about customers the agent should be notified about. It's sent after [`login`](#login).
```

Reply:
`incoming_customers` contain all the customers in agent's groups, even if the agent doesn't (and won't) follow. I think "aware of" is better in this case.

Reply:
Yeah, in that case, it's better. Or "know about".